config/routes.rb

```diff
@@ -19,6 +19,7 @@
 
     get '/accessibility', to: 'content#accessibility'
     get '/cookies', to: 'content#cookies_candidate', as: :cookies
+    get '/make-a-complaint', to: 'content#complaints', as: :complaints
```

Comment:
Is there any reason why we can’t make this a top-level path? It’s not a candidate-specific page, but applies to the whole service, Apply and Manage.

Reply:
All of our routes so far are namespaced (see the accessibility and cookies paths above), so I am keeping with that pattern.

But it is possible to make it top-level if we wanted.